appium/sample-scripts/python/testdroid_android.py

```diff
@@ -22,11 +22,17 @@ def setUp(self):
         ## You can set the parameters outside the script with environment variables.
         ## If env var is not set the string after or is used.
         ##
-        self.screenshotDir = os.environ.get('TESTDROID_SCREENSHOTS') or "/absolute/path/to/desired/directory"
+        self.screenshot_dir = os.environ.get('TESTDROID_SCREENSHOTS') or "/absolute/path/to/desired/directory"
         testdroid_url = os.environ.get('TESTDROID_URL') or "https://cloud.testdroid.com"
         testdroid_apiKey = os.environ.get('TESTDROID_APIKEY') or ""
         testdroid_app = os.environ.get('TESTDROID_APP') or ""
```

Comment:
os.environ.get('TESTDROID_APP') or "sample/BitbarSampleApp.apk" would work as default. Broker knows how to use this value.